controllers/control-plane/kyma_remote_sync_test.go

```diff
@@ -249,6 +249,18 @@ var _ = Describe("Kyma sync into Remote Cluster", Ordered, func() {
 			Should(Succeed())
 	})
 
+	It("Remote SKR Kyma get regenerated after it gets deleted", func() {
+		By("Delete SKR Kyma")
+		Eventually(DeleteCR, Timeout, Interval).
+			WithContext(ctx).
+			WithArguments(runtimeClient, remoteKyma).Should(Succeed())
+
+		By("Expect SKR Kyma get recreated")
+		Eventually(kymaExists, Timeout, Interval).
```

Comment:
this `kymaExits` need to be enhanced, it can be the situation that kyma is under deletion (with `DeletionTimestamp` exits), and it will return no error.

Two options:
1. add deletiontimestamp check in this kymaExists, but need to check if it fits for other tests which use this function.
2. write a dedicate function for this check.